src/main/java/org/mule/module/xml/internal/operation/SchemaValidatorOperation.java

```diff
@@ -147,13 +157,33 @@ private void trackError(SAXParseException exception) {
     });
   }
 
+  private boolean isBlank(String value) {
+    return value == null || value.trim().length() == 0;
+  }
+
+  private boolean isBlank(List<SchemaContent> list) {
+    return list == null || list.isEmpty();
+  }
+
+  private void checkSchemaInput(String schemas, List<SchemaContent> schemaContents) {
```

Comment:
Keep the Operation class as clean as possible with the method operation only. You should create another class with the responsibility of validate schemas. Could be a Service or a SchemaValidator, for instance. We would win an easier testability for all validation functionality without running a mulesoft environment.